packages/tools/src/tools/annotation/CircleROITool.ts

```diff
@@ -984,9 +981,19 @@ class CircleROITool extends AnnotationTool {
           modalityUnitOptions
         );
 
+        const invXRadiusSq =
+          ellipseObj.xRadius !== 0 ? 1 / ellipseObj.xRadius ** 2 : 0;
```

Comment:
You don't need to continue computing this here since it is done internally on the first call.